scripts/vulners.nse

```diff
@@ -0,0 +1,214 @@
+description = [[
+For each available cpe it prints the known vulns (links to the correspondent info).
+
+Its work is pretty simple:
+- work only when some software version is identified for an open port
+- take all the known cpe for that software (from the standard nmap output)
+- ask whether some known vulns exist for that cpe
```

Comment:
You probably want to mention here that it connects to a remote service.

Reply:
Definitely yes. Will refactor the description.